xml/ay_custom_user_scripts.xml

```diff
@@ -402,14 +405,14 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
         <entry>
          <para>
           Specify the interpreter that must be used for the script.
-          Supported options are <literal>shell</literal> and
-          <literal>perl</literal>.
+          Any interpreter available at given environment is possible
+          to specify. Full path even with parameters is possible.
```

Comment:
You are losing the documentation of the `shell` and `perl` special values. Also, a `python` special value has been supported by the code.

Reply:
well, I see it as a bit deprecated less powerful options, but I can add back those three specific values and mention that it is supported by debug flag?

Reply:
As they are valid, I guess we should keep them in the documentation. Perhaps we can describe them in a `<important/>` section as deprecated.